src/components/_templates/Orders/OrderList/OrderList.tsx

```diff
@@ -0,0 +1,63 @@
+import { List, Select } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { CONSTANTS_TEXT, SELECT_OPTIONS } from 'src/constants'
+import { getStatusValue } from 'src/helper/helper'
+import { IOrder } from 'src/redux'
+import { useAppDispatch, useAppSelector } from 'src/redux/hooks'
+import { requestOrdersInfo } from 'src/redux/orders/actions'
+import { getOrders } from 'src/redux/orders/selectors'
+import { Title } from 'src/typography'
+import { AddOrders } from '../AddOrders'
+import { OrdersDetails } from '../OrdersDetails'
+import './OrderList.scss'
+
+export const OrderList = () => {
+  const [currentStatus, setCurrentStatus] = useState(['in storage'])
```

Comment:
are you sure that you want to use array here

Reply:
> are you sure that you want to use array here

well, yes, I can pass several parameters here, and then such statuses will be displayed on the page